### PR TITLE
[PE-4544] Added bash script to automate classname replacement

### PR DESCRIPTION
**Pull request by janibolkvadze**
@dani-cl-madrid @mattnickles @jllr 

https://ctl.atlassian.net/browse/PE-4544

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0e7f2aa4f9c267d7eb0c212689adc6b77ba16b42-chi-pr.enterprise.ctl.io" target="_blank"> https://0e7f2aa4f9c267d7eb0c212689adc6b77ba16b42-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/814/ for more details.

File: scripts/chi-classnames/replacement.sh [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    
+)
+
+for className in "${ARRAY[@]}" ; do
+    regex='\([^a-zA-Z0-9]\)'"${className%%:*}"'\([^a-zA-Z0-9]\)'
+    change="s/${regex}/\1${className##*:}\2/g"
+    regexnl='\([^a-zA-Z0-9]\)'"${className%%:*}"'$'
+    changenl="s/${regexnl}/\1${className##*:}/g"
+
+    find path/to/project/folder -type f -exec sed -i '' "${change};${changenl}" '{}' +
```
Comment (dani-cl-madrid) [inline scripts/chi-classnames/replacement.sh]: Please, put path/to/project/folder in a constant on top of the file, so it is easy to find and configure.

Comment (janibolkvadze) [reply scripts/chi-classnames/replacement.sh]: There will be an alternative .bat script for windows users. Thanks for the suggestions!

File: scripts/chi-classnames/replacement.sh [hunk 2]
```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+pathtofolder='path/to/project/folder'
```
Comment (dani-cl-madrid) [inline scripts/chi-classnames/replacement.sh]: Please, use uppercase for constants. I.e PROJECT_SRC or SRC or PATH_TO_PROJECT_FILES...

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://890f99d665213228039828569f35946856aaa704-chi-pr.enterprise.ctl.io" target="_blank"> https://890f99d665213228039828569f35946856aaa704-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/816/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f2e87e5334edf41f56efe0ff07501ad638b60c34-chi-pr.enterprise.ctl.io" target="_blank"> https://f2e87e5334edf41f56efe0ff07501ad638b60c34-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/817/ for more details.

**Comment by jllr** [comment]
@dani-cl-madrid can you review if everything is ok and your suggestions have been included? Thanks in advance

File: scripts/chi-classnames/replacement.sh [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    "m-popover:chi-popover"
+
+    # Price
+    "a-price:chi-price"
+
+    # Radio
+    "a-radio:chi-radio"
+
+    # Range slider
+    "a-rangeSlider:chi-rangeSlider"
```
Comment (jllr) [inline scripts/chi-classnames/replacement.sh]: @janibolkvadze can you change chi-rangeSlider to chi-range-slider?

File: scripts/chi-classnames/replacement.sh [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    "-notFluid:-no-fluid"
+    "-hasLabel:-label"
+    "-floatingLabel:-floating-label"
+    "-horizontalLabels:-horizontal-label"
+    "-nostyle:-no-style"
+    "-activeAlt:-active-alt"
+    "-disableScroll:-disable-scroll"
+    "-hasActive:-has-active"
+    "a-sliding-border:chi-sliding-border"
+    "m-example:chi-example"
```
Comment (jllr) [inline scripts/chi-classnames/replacement.sh]: @janibolkvadze please add "a-arrow:chi-arrow"

File: scripts/chi-classnames/replacement.sh [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    "a-collection__title:chi-collection__title" # Docu only
```
Comment (jllr) [inline scripts/chi-classnames/replacement.sh]: @janibolkvadze please add "-bgInverse:-bg--inverse"

File: scripts/chi-classnames/replacement.sh [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+    # Label
+    "a-label:chi-label"
+
+    # Modal
+    "a-modal:chi-modal"
+
+    # Pagination
+    "a-pagination:chi-pagination"
+
```
Comment (jllr) [inline scripts/chi-classnames/replacement.sh]: @janibolkvadze please add Picker component changes "m-pickerGroup:chi-picker-group", "m-picker:chi-picker__item", "a-picker:chi-picker"

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://26ac175e7ade901eb165c0040ab7aef660b8d6b5-chi-pr.enterprise.ctl.io" target="_blank"> https://26ac175e7ade901eb165c0040ab7aef660b8d6b5-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/853/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://02a6b2aba94df6e03698e6f88a92fd31c1c3a8dd-chi-pr.enterprise.ctl.io" target="_blank"> https://02a6b2aba94df6e03698e6f88a92fd31c1c3a8dd-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/854/ for more details.

**Comment by jllr** [comment]
This PR is dependant of #560 changes in heading text utility classes

**Comment by mattnickles** [comment]
@jllr @janibolkvadze #560 has been merged

**Comment by jllr** [comment]
@janibolkvadze 
"a-h1:chi-h1"
"a-h2:chi-h2"
"a-h3:chi-h3"
"a-h4:chi-h4"
"a-h5:chi-h5"
"a-h6:chi-h6"

this has to change to:

"a-h1:-text--h1"
"a-h2:-text--h2"
"a-h3:-text--h3"
"a-h4:-text--h4"
"a-h5:-text--h5"
"a-h6:-text--h6"

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://1604fe4816de1553bfa6a7901476a171d2712c6b-chi-pr.enterprise.ctl.io" target="_blank"> https://1604fe4816de1553bfa6a7901476a171d2712c6b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/865/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://292a45ad99dc04611af23326545fd6e3b6e1cf12-chi-pr.enterprise.ctl.io" target="_blank"> https://292a45ad99dc04611af23326545fd6e3b6e1cf12-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/908/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://696ca4d4ecfef5f6aefda48e1da2b9ea455cce44-chi-pr.enterprise.ctl.io" target="_blank"> https://696ca4d4ecfef5f6aefda48e1da2b9ea455cce44-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/917/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f2bcf2e934239df42c5976ed8adb899bec07be60-chi-pr.enterprise.ctl.io" target="_blank"> https://f2bcf2e934239df42c5976ed8adb899bec07be60-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/920/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-b63qpc, https://ops.elasticbox.com/services/instances/i-nrd6lr, https://ops.elasticbox.com/services/instances/i-77sb9a, https://ops.elasticbox.com/services/instances/i-rstb7s, https://ops.elasticbox.com/services/instances/i-mppkz8, https://ops.elasticbox.com/services/instances/i-vth7zs, https://ops.elasticbox.com/services/instances/i-axds3q, https://ops.elasticbox.com/services/instances/i-fdniio, https://ops.elasticbox.com/services/instances/i-vv82dc

**Pull request merged**